cpp/test/Slice/errorDetection/NestedModule.err

```diff
@@ -0,0 +1,10 @@
+NestedModule.ice:9: illegal identifier `Holder': `Holder' suffix is reserved
+NestedModule.ice:11: illegal identifier `Holder': `Holder' suffix is reserved
+NestedModule.ice:15: illegal leading underscore in identifier `__Iceberg'
+NestedModule.ice:18: illegal identifier `Holder': `Holder' suffix is reserved
+NestedModule.ice:20: illegal identifier `Holder': `Holder' suffix is reserved
+NestedModule.ice:24: illegal identifier `APtr': `Ptr' suffix is reserved
+NestedModule.ice:24: illegal identifier `BPrx': `Prx' suffix is reserved
+NestedModule.ice:24: illegal identifier `CHelper': `Helper' suffix is reserved
+NestedModule.ice:27: missing identifier: illegal empty identifier
+NestedModule.ice:27: illegal identifier `Helper': `Helper' suffix is reserved
```

Comment:
This will have to be updated accordingly if you implement my above suggestion.

Reply:
I don't find "illegal empty identifier" as a good way to say "a module name must start with a letter".

Reply:
The message became `illegal identifier: module identifiers cannot start with '::' prefix`